typescript/packages/browser/README.md

```diff
@@ -1,6 +1,11 @@
 ## @0x/mesh-browser
 
-TypeScript and JavaScript bindings for running Mesh directly in the browser.
+This package provides an easy way to run a browser-based mesh node. Specifically, it
+provides Typescript and Javascript bindings that can be used to interact with a mesh
```

Comment:
nit: Mesh should be capitalized unless we are referring specifically to something that is not capitalized (e.g. the name of a Docker image or executable).

```suggestion
provides Typescript and Javascript bindings that can be used to interact with a Mesh
```